activefires_pp/spatiotemporal_alarm_filtering.py

```diff
@@ -84,6 +85,10 @@ def __init__(self, configfile):
 
         self.sos_alarms_file_pattern = self.options['geojson_file_pattern_alarms']
         self.restapi_url = self.options['restapi_url']
+        self.restapi_xauth = get_xauth_environment_variable()
+        if self.restapi_xauth is None:
+            raise OSError("Environment variable XAUTH_SMHI_FIREALARMS_REST_API not set!")
```

Comment:
This should go in `get_xauth_environment_variable`

Reply:
Fixed